android/src/main/java/com/it_nomads/fluttersecurestorage/FlutterSecureStoragePlugin.java

```diff
@@ -59,18 +59,14 @@ public void initInstance(BinaryMessenger messenger, Context context) {
      * live devices in release mode.
      * The most convenient place for that appears to be onMethodCall().
      */
-    private void ensureInitStorageCipher() {
-        if(storageCipher == null) { //Check to avoid unnecessary entry into syncronized block
```

Comment:
@wytesk133 according to the article, wouldn't making storageCipher `volatile` be better fix? 
Without synchronization it's seems race condition could break entire storage due to different aes keys.